jax/config.py

```diff
@@ -59,11 +59,10 @@ def __init__(self):
     self.FLAGS = NameSpace(self.read, self.update)
     self.use_absl = False
     self._contextmanager_flags = set()
+    self._update_hooks = {}
 
     # TODO(mattjj): delete these when only omnistaging is available
     self.omnistaging_enabled = bool_env('JAX_OMNISTAGING', True)
```

Comment:
Should we delete this flag as well?

Reply:
I hesitate to lump that in here because we might have to change downstream (e.g. google-internal) callers that rely on the flag existing, including perhaps its value being set by this environment variable and then read by their own code. I'd rather leave that for follow-up work. (If we were somehow 100% sure we could delete this without having to make downstream changes as a result, then I'd like to as well!)

Reply:
OK, makes sense!

Reply:
To be clear, I agree with your suggestion, just that I might sequence it in an immediate follow-up PR!